backend/src/schema/resolvers/users.spec.js

```diff
@@ -882,27 +893,27 @@ describe('DeleteUser', () => {
             })
           })
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-26T11:47:15Z" title="Wednesday, February 26th 2020, 12:47:15 pm +01:00">Feb 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -542,6 +542,12 @@ describe('DeleteUser', () => {
               mutate({ mutation: deleteUserMutation, variables }),
             ).resolves.toMatchObject(expectedResponse)
           })
+
+          it('deletes user avatar and post hero images', async () => {
+            await expect(neode.all('Image')).resolves.toHaveLength(22)
+            await mutate({ mutation: deleteUserMutation, variables })
+            await expect(neode.all('Image')).resolves.toHaveLength(20)
+          })
```


where are these magic numbers coming from?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-04T13:10:20Z" title="Wednesday, March 4th 2020, 2:10:20 pm +01:00">Mar 4, 2020</time>_

---


Well, I don't know myself, I just ran the test and checked how many images are set up in the `beforeEach`s prior to the test case :point_up: then I wanted it to remove more than 1 image.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-11T19:27:20Z" title="Wednesday, March 11th 2020, 8:27:20 pm +01:00">Mar 11, 2020</time>_

---


humm... don't really like that